### PR TITLE
Clarify aspects of the server push extension

### PR DESCRIPTION
**Pull request by pgjones**
See commit messages

@jlaine look good to you?

### CODE REVIEW

**Comment by andrewgodwin** [comment]
Seems good to me. Waiting for Jeremy to comment before I merge.

File: docs/extensions.rst [hunk 1]
```diff
@@ -70,6 +70,9 @@ Keys:
...
 
 The ASGI server should then attempt to send a server push (or push
-promise) to the client. If the client accepts, the server should create
-a new connection to a new instance of the application and treat it as
-if the client had made a request.
+promise) to the client. If the client supports server push, the server
+should create a new connection to a new instance of the application
+and treat it as if the client had made a request.
+
+The ASGI server should set the pseudo authority header value to be the
```
Comment (jlaine) [inline docs/extensions.rst]: Nit: "should set the `:authority` pseudo-header" might be clearer?

Comment (pgjones) [reply docs/extensions.rst]: ok

**Comment by pgjones** [comment]
@jlaine good to merge now?

**Comment by pgjones** [comment]
Thank you :)

**Comment by andrewgodwin** [comment]
Thank _you_! It's small things like this that I would never have caught myself.

**Pull request merged**